libs/execution_context/ebpf_native.c

```diff
@@ -685,6 +690,11 @@ _ebpf_native_create_maps(_Inout_ ebpf_native_module_t* module)
     if (module->maps == NULL) {
         EBPF_RETURN_RESULT(EBPF_NO_MEMORY);
     }
+
+    for (uint32_t count = 0; count < map_count; count++) {
+        module->maps[count].handle = ebpf_handle_invalid;
+    }
```

Comment:
Maybe we can move this code inside `_ebpf_native_initialize_maps`. `_ebpf_native_initialize_maps` is also setting the map handles to `ebpf_handle_invalid`, but there is a possibility that the `_ebpf_native_initialize_maps` may bail before setting all the handles.

So we can instead fix it there (in `_ebpf_native_initialize_maps`)